docs/developer/release/qa.md

```diff
@@ -1,4 +1,4 @@
-## QA
+# QA
```

Comment:
Maybe we should have some kind of introduction here, listing the large QA docs, automated scripts and manual steps, to give the reader some context.

Reply:
You have a point. I tried to add a bit more context in a73a03e6.